### PR TITLE
Update nccr-mediality.csl

### PR DESCRIPTION
**Pull request by cnotz**
Short citations without italics (as in stylesheet) and prefix for second citations.

### CODE REVIEW

File: nccr-mediality.csl [hunk 1]
```diff
@@ -311,6 +311,7 @@
         <else-if position="subsequent">
           <text macro="contributors-short" suffix=", "/>
           <text macro="title-short"/>
+          <text variable="first-reference-note-number" prefix=" (Anm. " suffix=")"/>
```
Comment (rmzelle) [inline nccr-mediality.csl]: @cnotz, just curious, but what does "Anm" stand for here?

@adam3smith, do you know if we have a CSL term for this?

**Comment by adam3smith** [comment]
"Anm." stands for "Anmerkung," i.e. "note." No CSL term, though I think there should be (we [hardcode the equivalent](https://github.com/citation-style-language/styles/blob/master/oscola.csl#L594) in English styles, too).

File: nccr-mediality.csl [hunk 2]
```diff
@@ -174,7 +174,7 @@
     <group delimiter=", ">
       <choose>
         <if type="book">
-          <text variable="title" form="short" font-style="italic"/>
+          <text variable="title" form="short"/>
```
Comment (adam3smith) [inline nccr-mediality.csl]: give this, let's simplify the macro. You can reduce it to a single line, i.e. `<text variable="title" form="short"/>`

**Comment by adam3smith** [comment]
One small nit above. Everything else looks good. Thanks

**Comment by rmzelle** [comment]
> "Anm." stands for "Anmerkung," i.e. "note." No CSL term, though I think there should be (we hardcode the equivalent in English styles, too).

There is a "note" term, though. (see e.g. https://github.com/citation-style-language/locales/blob/020712589f08bd74adf53c99f808fbbefebd463f/locales-en-US.xml#L191)

**Comment by cnotz** [comment]
Thank you for your critical eyes and your help simplificating this style!

File: nccr-mediality.csl [hunk 3]
```diff
@@ -310,8 +300,9 @@
         </else-if>
         <else-if position="subsequent">
           <text macro="contributors-short" suffix=", "/>
-          <text macro="title-short"/>
-          <text variable="first-reference-note-number" prefix=" (Anm. " suffix=")"/>
+          <text variable="title" form="short"/>
+          <text term="note" form="short" prefix=" (" suffix="."/>
+          <text variable="first-reference-note-number" prefix=" " suffix=")"/>
```
Comment (rmzelle) [inline nccr-mediality.csl]: @cnotz, it's better to replace

```xml
          <text term="note" form="short" prefix=" (" suffix="."/>
          <text variable="first-reference-note-number" prefix=" " suffix=")"/>
```

by

```xml
        <group prefix=" (" suffix=")" delimiter=" ">
          <text term="note" form="short"/>
          <text variable="first-reference-note-number"/>
        </group>
```

and change

```
<term name="note" form="short">Anm</term>
```

by

```
<term name="note" form="short">Anm.</term>
```

That clarifies how the parentheses belong together (it's a good habit to use groups wherever possible in styles for placing punctuation). (this `<group/>` would also prevent the "note" term from being printed if the "first-reference-note-number" variable doesn't have a value, although I guess that shouldn't really be possible for `position="subsequent"`)

In our world of CSL we also always define abbreviated terms with periods.

**Comment by adam3smith** [comment]
@rmzelle I put in the changes you suggested, go ahead and merge if you're good with this otherwise.

**Comment by rmzelle** [comment]
Great, thanks.

**Comment by cnotz** [comment]
Thank you so much for your work!
> Am 21.10.2016 um 16:52 schrieb csl-bot <notifications@github.com>:
> 
> 😟

**Pull request merged**